### PR TITLE
feat: add helper for starting kratos e2e

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: cmd/identities/helpers_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	require.Len(t, stdErr, 0, stdOut)
+	return stdOut
+}
+
+func execErr(t *testing.T, cmd *cobra.Command, args ...string) string {
+	stdOut, stdErr, err := exec(cmd, nil, args...)
+	require.True(t, errors.Is(err, cmdx.ErrNoPrintButFail))
+	require.Len(t, stdOut, 0, stdErr)
+	return stdErr
+}
```
Comment (zepatrik) [inline cmd/identities/helpers_test.go:63]: This part is now part of `cmdx`, maybe remove it in this go? I can also do that btw

Comment (aeneasr) [reply cmd/identities/helpers_test.go:63]: Ah yeah makes sense, I first have to wrap everything in new* funcs to make it pass. We can then do another pass and remove these helpers and move to ory/x. Would be great if you could do that. I think it makes sense to do it once this PR is merged

Comment (zepatrik) [reply cmd/identities/helpers_test.go:63]: They are already in `ory/x` 1:1, you can just remove them here and import instead. But we can also separate that.

Comment (aeneasr) [reply cmd/identities/helpers_test.go:63]: The method signatures are a bit different so I'd separate it

**Pull request merged**